src/components/core/PivotTable.tsx

```diff
@@ -1127,9 +1105,13 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
 
         invariant(execution !== undefined, "changing column width prior execution cannot work");
 
-        columns.forEach(column => {
-            this.addToManuallyResizedColumn(column);
-        });
+        if (this.isCtrlOrMetaKeyPressed() && columns.length === 1 && isMeasureColumn(columns[0])) {
```

Comment:
the condition is repeated from the method above, maybe it could be somehow extracted and named

Reply:
done